### PR TITLE
Day5

### PR DESCRIPTION
**Pull request by kghite**

### CODE REVIEW

File: day5/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Chapter 13
+
+## 13.4
+
+Fill in the body of `findEntry(Object target)`
+Fill in the body of `put(K key, V value)`
+Fill in the body of `remove(Object key)`
```
Comment (thecardkid) [inline day5/README.md]: Can you update the readme to specific which class you're supposed to make changes to?

File: solutions/src/com/allendowney/javacs/MyBetterMap.java [hunk 2]
```diff
@@ -66,18 +66,14 @@ public void clear() {
 	@Override
```
Comment (thecardkid) [inline solutions/src/com/allendowney/javacs/MyBetterMap.java]: Not sure why changes to solutions/ are added to this commit. Can you undo these changes / create a new commit without these changes?

**Comment by davidabrahams** [comment]
@thecardkid I fixed the readme. I'm good to merge
